smarty/sampler_elemental.py

```diff
@@ -88,13 +88,13 @@ def __init__(self, molecules, basetypes_filename, initialtypes_filename, decorat
         This is just a proof of concept.  No scoring of molecular properties is performed.
 
         """
-        
-        
+
+
```

Comment:
I have my vim settings delete extra spaces at the end of a line. 
I'm assuming whoever edited this last did not have those settings in place...